.gitignore

```diff
@@ -2,6 +2,9 @@
 coverage/*
 logs/*
 node_modules/*
+sitemap.xml
+sitemap-*.xml
+sitemap-*.xml.gz
```

Comment:
This can be removed I think

Reply:
What if someone (like me;) ) accidentally generates sitemaps locally and pushes to github? It will override the files generated by the github action.

Reply:
On second thought, even github action won't be able to commit with these entries. Removing now

Reply:
Or we can also put git add --force in the workflow and keep these entries in gitignore. WDYT?

Reply:
I would remove it from .gitignore to be on the save side.

Reply:
fixed